ui-components/src/global/vendor/ag_grid.scss

```diff
@@ -119,6 +119,13 @@
   @apply bg-gray-200;
 }
 
+.sticky {
+  position: fixed;
```

Comment:
nit: should we be using tailwind styles here? (Fine if not, since this is pretty specific)

Reply:
Done, thanks for pointing that out!